### DISCUSSION TITLE
[issue] Filter localization

### DISCUSSION THREAD

**Discussion by rusaym** [mui/mui-x#1012] [Archive]
Can I somehow localize these filter props?
![1](https://user-images.githubusercontent.com/17137916/107209993-995f9380-6a14-11eb-901d-0fb9f9e98b9f.png)

**Comment by oliviertassinari** [comment]
@DanailH solve the problem in https://github.com/mui-org/material-ui-x/pull/983

**Comment by DanailH** [comment]
This should be now available or be available in the next release. Can we close this discussion if everyone agrees?

**Reply by oliviertassinari** [reply]
I can't see where a discussion can be closed :/

@dtassone What do you think about disabling the GitHub discussion tab? It seems more harmful than helpful.

We have to look into two different places. People most often report problems that can be promoted as github issues. Discussions can happen in issues, it seems fine.
